assets/scss/_home.scss

```diff
@@ -247,6 +248,293 @@
+            right: 0;
+        }
+        .bubble-lg {
+            max-width: 44vw;
+            max-height: 44vw;
+            width: 50%;
+            height: 50%;
+            padding: 20px;
+            top: 400px;
+            left: 5%;
+        }
+        .bubble-md {
+            max-width: 35vw;
+            max-height: 35vw;
+            width: 38%;
+            height: 38%;
+            padding: 0;
+            top: 10px;
+            left: 0;
+        }
+        .bubble-sm {
+            max-width: 28vw;
+            max-height: 28vw;
+            width: 30%;
+            height: 30%;
+            padding: 10px;
+            top: 534px;
+            right: 10%;
+        }
+        .bubble-xs {
+            max-width: 139px;
+            max-height: 139px;
+            width: 80%;
+            height: 80%;
+            padding: 20px;
+            top: 760px;
+            left: 48%;
+        }
+        .bubble-xxs {
+            width: 80px;
+            height: 80px;
+            padding: 20px;
+            top: 940px;
+            left: 55%;
+        }
+        .deepsea-sub {
+            position: absolute;
+            bottom: 0;
+            left: 50%;
+            margin-left: -165px;
+        }
+        @media (max-width: 670px) {
+            // Mobile layout for Make Games section
+            .float, .floatslow {
+                animation: none;
+            }
+            .bubble-container {
+                height: auto;
+            }
+            .bubble, .bubble-empty.bubble-xs {
+                background: none;
+                position: relative;
+                max-width: unset;
+                max-height: unset;
+                width: auto;
+                height: auto;
+                padding: 30px 10px;
+                blockquote p {
+                    font-size: 16px;
+                    line-height: 24px;
+                }
+                figcaption {
+                    font-size: inherit;
+                    line-height: inherit;
+                }
+            }
+            .bubble-xl, .bubble-lg, .bubble-md, .bubble-sm, .bubble-xs {
+                left: auto;
+                top: auto;
+                right: auto;
+            }
+            .bubble-empty.bubble-xxs {
+                position: relative;
+                top: auto;
+                right: auto;
+                left: -14px;
+                margin-left: auto;
+                margin-right: auto;
+                margin-bottom: 20px;
+            }
+            .deepsea-sub {
+                position: relative;
+                left: auto;
+            }
+        }
+
+        @media (min-width: 768px) {
+            p:not(.panel-body p, blockquote p) {
+                font-size: 21px;
+            }
+
+            h4:not(.panel h4) {
+                font-size: 30px;
+            }
+            .panel-text {
+                width: 60%;
+            }
+            .panel-img {
+                width: 40%;
```

Comment:
The Figma mockup has `.panel-img` wider and `.panel-text` narrower. At Desktop-Huge the Figma mockup has the image extending nearly to the right side of Victor's portrait. Since the image for Sue contains text, and the text is more readable at a larger size, I think this should be changed.

Reply:
done